src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -744,7 +744,8 @@ private static string RetrieveProcessUserName(Process process)
             try
             {
                 int error;
-                if (!Win32Native.OpenProcessToken(process.Handle, TOKEN_QUERY, out processTokenHandler)) { return null; }
+                if (!Win32Native.OpenProcessToken(process.Handle, TOKEN_QUERY, out processTokenHandler))
+                { return null; }
```

Comment:
Too many unrelated changes in the PR. Please revert or move to another PR.

Reply:
Yea sorry that’s vscode autoformating files on save. I’ll revert them before making the PR ready.

Reply:
These have been reverted with the latest commit.